src/app.js

```diff
@@ -0,0 +1,64 @@
+import TodoCount from './TodoCount.js';
+import TodoList from './TodoList.js';
+import TodoInput from './TodoInput.js';
+import TodoFilter from './TodoFilter.js';
+import {FilterOptions} from './constants.js';
+
+function App() {
+  const $todoInput = document.querySelector('#new-todo-title');
+  const $todoList = document.querySelector('#todo-list');
+  const $todoCount = document.querySelector('.todo-count');
+  const $todoFilter = document.querySelector('.filters');
+  this.data = [];
+  this.filteredData = [];
+  this.selected = 'all';
+
+  this.addItem = (text) => {
+    this.data.push({
+      text: text,
+      isCompleted: false,
+    });
+
+    this.filterItems(this.selected);
+  };
+
+  this.removeItem = (index) => {
+    this.data.splice(index, 1);
+
+    this.filterItems(this.selected);
+  };
```

Comment:
```suggestion
  this.removeItem = filteredIndex => {
    const item = this. filteredData[filteredIndex];
    const index = this.data.find(v => v === this.get)
    this.data.splice(index, 1);
    this.filterItems(this.selected);
  };
```
index가 다르더라도 실제 객체는 똑같은걸 공유하기 때문에 이런식으로 작성해주면 된답니다!

Reply:
아하.. 코드 참고해서 수정하고 있는데 객체 내용이 똑같더라도 원하는 아이템을 잘 삭제해주는게 신기하네요👍

Reply:
혹시 27번줄의 `this.get`은 js의 어떤 기능이 있는 코드인가요? 검색해도 나오는 것 같진 않아서 참고로 작성해주신 것이라 생각하고 `const index = this.data.findIndex(v => v === item)`으로 수정했습니다 ㅎㅎ;

Reply:
isCompleted는 다르지만 내용은 동일한 다른 index값을 삭제할 수도 있지 않을까요??

Reply:
@wooooooood 앗 this.get은 제가 잘못 작성한 것 같아요 ㅠㅠ item이 맞습니다!

Reply:
``js
  this.removeItem = filteredIndex => {
    const item = this. filteredData[filteredIndex];
    const index = this.data.findIndex(v => v == item)
    this.data.splice(index, 1);
    this.filterItems(this.selected);
  };
```
제가 졸면서 썼느지 .. 잘못된 부분이 몇 개 있네요 ㅠㅠ 이런식으로 해줘야합니다!

Reply:
남겨주신 리뷰를 잘못 이해했었네요. 정말 고유아이디가 필요없겠어요. 잘 배웠습니다. :+1: